src/nkr_proxy/cron/check_session_expire.py

```diff
@@ -247,17 +247,69 @@ def check_and_close_expired_sessions(session_max_age_seconds):
 
                 stats.only_some_closed = True
 
+
     stats.log_stats()
 
     logger.info('---------------------------------------------------------------//')
 
 
+def check_and_close_expired_active_sessions(max_seconds_after_user_first_active):
+    """
+    After maximum duration of REMS application it is closed regardless of user activity.
+    """
+
+    stats = Stats()
+    logger.info('Begin checking active expired sessions...')
+    
+    for key in cache.scan_iter('user-first-active:*'):   
+        
+        user_id = key.decode('utf-8').split('user-first-active:')[1]
+        user_first_active_ts = round(float(cache.get(key).decode('utf-8')))
```

Comment:
Would it possible to make a sub-function from these two lines and use it in both - active and inactive session expire? (optional)